frontend/src/utils/pluralize.ts

```diff
@@ -56,6 +56,32 @@ const plurals = {
       id: "pages.plural.other",
     }),
   },
+  Pages: {
```

Comment:
Can we just reuse `pages` and capitalize with CSS?

Reply:
I think so? Though in english, articles like "as" shouldn't be capitalized in title case, which setting `text-transform: capitalize` would do, so I'd prefer not to. I'll update "as" to be lowercase though!

Reply:
I'd vote to just lowercase it all, it already reads like a sentence and personally I find it harder to read longer title cased text...

Reply:
Sure, works for me!